ndscan/plots/image_2d.py

```diff
@@ -227,8 +230,13 @@ def bounds(schema):
 
         image_item = pyqtgraph.ImageItem()
         self.addItem(image_item)
-        self.plot = _ImagePlot(image_item, self.data_names[0], *bounds(self.x_schema),
-                               *bounds(self.y_schema), hints_for_channels)
+        colorbar = self.getPlotItem().addColorBar(image_item,
+                                                  colorMap=colormaps.plasma,
+                                                  values=(0, 1),
```

Comment:
What is the significance of passing `(0, 1)` here? (I assume this is overridden using `setLevels` later?)

Reply:
No significance, and we are overriding, so will remove. In fact, the same applies to the `colorMap=...`